caddyfile/dispenser.go

```diff
@@ -204,6 +204,13 @@ func (d *Dispenser) SyntaxErr(expected string) error {
 	return errors.New(msg)
 }
 
+// RewritePathErr returns an error indicating the path being rewritten to
+// is missing an initial /
+func (d *Dispenser) RewritePathErr() error {
```

Comment:
hmm, well all the other parsing/syntax errors seem to be raised in dispenser.go.  

I can use `d.Err()` but I don't get access to `d.Val()` which would be nice.  I can just raise a syntax error if we don't want new errors created.

>  Syntax error: Unexpected token 'wpadmin/admin.php', expecting '/wpadmin/admin.php'

but IMHO it is a little too subtle better to tell the user exactly what the issue is.

Could I create another general purpose method to raise an error that includes `d.Val()` as a parameter?  I will see if i can get the output correct.